commands/manifest/template.go

```diff
@@ -118,3 +109,50 @@ func templateCmd(c *cli.Context) error {
 
 	return nil
 }
+
+func GetTemplatesManifests(m dx.Manifest) (string, error) {
+
+	switch {
+	case m.Manifests != "" && m.Chart.Name != "":
+		templatesManifests, err := TemplateChart(m)
+		if err != nil {
+			fmt.Errorf("cannot template Helm chart %s", err)
+		}
+
+		templatesManifests += m.Manifests
+
+		return templatesManifests, err
+	case m.Chart.Name != "":
+		templatesManifests, err := TemplateChart(m)
+		if err != nil {
+			fmt.Errorf("cannot template Helm chart %s", err)
+		}
+
+		return templatesManifests, err
+	case m.Manifests != "":
+		return m.Manifests, nil
+	default:
```

Comment:
This case should result in validation failure. We should instruct the user that either of one is mandatory.